front/lib/models/workspace.ts

```diff
@@ -24,6 +24,7 @@ export class Workspace extends Model<
   declare sId: string;
   declare name: string;
   declare description: string | null;
+  // TODO(2024-02-09 flav) Remove.
```

Comment:
Why not remove right away?

Reply:
Since the last incident on the remove column. We don't trust anymore Sequelize so we delete the column manually.